### DISCUSSION TITLE
Woff2 format

### DISCUSSION THREAD

**Discussion by eighty4** [fontsource/fontsource#794] [Q&A]
Do fontsource npm packages get bundled with an optimized woff2 file?

I've been downloading woff from Google fonts and converting to woff2. I didn't see any mention of this anywhere in the docs.

**Comment by ayuhito** [comment]
Yes, all Fontsource packages automatically default to the WOFF2 format. You can verify all the files in the Github repository e.g. [Inter](https://github.com/fontsource/font-files/tree/main/fonts/google/inter/files).
